diff --git a/dist/uni-h5.es.js b/dist/uni-h5.es.js
--- a/dist/uni-h5.es.js
+++ b/dist/uni-h5.es.js
@@ -23410,7 +23410,7 @@ function useShowTabBar(emit2) {
   const tabBar2 = useTabBar();
   const showTabBar2 = computed(() => route.meta.isTabBar && tabBar2.shown);
   updateCssVar({
-    "--tab-bar-height": tabBar2.height
+    "--tab-bar-height": tabBar2?.height || 0
   });
   return showTabBar2;
 }
